homeassistant/components/fan/isy994.py

```diff
@@ -13,9 +13,13 @@
 import homeassistant.components.isy994 as isy
 from homeassistant.const import STATE_UNKNOWN, STATE_ON, STATE_OFF
 from homeassistant.helpers.typing import ConfigType
+from homeassistant.helpers.entity import ToggleEntity
 
 _LOGGER = logging.getLogger(__name__)
 
+SPEED_MEDIUM = 'med'
```

Comment:
Don't overwrite this constant. If you want to have a constant for `'med'`, name it something specific for ISY, eg ISY_MEDIUM_SPEED.